code/modules/awaymissions/mobspawn.dm

```diff
@@ -0,0 +1,32 @@
+/obj/effect/landmark/corpse/mob_spawn
+	density = 1
+	anchored = 1
+
+/obj/effect/landmark/corpse/mob_spawn/attack_ghost(mob/user)
+	if(ticker.current_state != GAME_STATE_PLAYING)
+		return
+	var/ghost_role = alert("Become [mobname]? (Warning, You can no longer be cloned!)",,"Yes","No")
```

Comment:
Thats intentional, look up the format for alert - there is additional info you could add there, but makes no sense to in this instance.